sdk/python/v1beta1/kubeflow/katib/constants/constants.py

```diff
@@ -56,3 +58,26 @@
 BASE_IMAGE_MXNET = "docker.io/mxnet/python:1.9.1_native_py3"
 
 DEFAULT_DB_MANAGER_ADDRESS = "katib-db-manager.kubeflow:6789"
+
+# Constants for Tune API.
+STORAGE_INITIALIZER = "storage-initializer"
+# The default value for dataset and model storage PVC.
+PVC_DEFAULT_SIZE = "10Gi"
+# The default value for PVC access modes.
+PVC_DEFAULT_ACCESS_MODES = ["ReadWriteOnce", "ReadOnlyMany"]
+
+STORAGE_INITIALIZER_IMAGE = "docker.io/kubeflow/storage-initializer"
+
+STORAGE_INITIALIZER_VOLUME_MOUNT = client.V1VolumeMount(
+    name=STORAGE_INITIALIZER,
+    mount_path=INIT_CONTAINER_MOUNT_PATH,
+)
+
+STORAGE_INITIALIZER_VOLUME = client.V1Volume(
+    name=STORAGE_INITIALIZER,
+    persistent_volume_claim=client.V1PersistentVolumeClaimVolumeSource(
+        claim_name=STORAGE_INITIALIZER
+    ),
+)
+
+TRAINER_TRANSFORMER_IMAGE = "" # Need to be built using the `trainer` file
```

Comment:
Based on the [trainer](https://github.com/kubeflow/training-operator/tree/master/sdk/python/kubeflow/trainer) in the Training Operator, I updated the Python script to properly process the parameters and obtain metrics for hyperparameter optimization. Therefore, we need to build a new image for `TRAINER_TRANSFORMER_IMAGE` based on the updated code for the `tune` API. Currently, this is left blank. @andreyvelich Can you guide me on how to push the image to Kubeflow's repository?